Tarea_11/Dario-Sotelo/Dario/index.js

```diff
@@ -0,0 +1,26 @@
+// Async await
+const { default: axios } = require("axios");
+
+const API = "https://pokeapi.co/api/v2/pokemon";
+
+async function main() {
+  const response = await axios.get(API);
+  console.log(response.data);
+  const arrayPromises = response.data.results.map((pokemon) => {
+    return axios.get(`${API}/${pokemon.name}`);
+
```

Comment:
creo que te faltó cerrar el código.
```suggestion
  const arrayPromises = response.data.results.map((pokemon) => {
    return axios.get(`${API}/${pokemon.name}`);
}
```